GetStream/stream-chat-swift
# Add IDs To generated referenceDocs

# Magic link
https://stream-ids-implementation.loca.lt/chat/docs/sdk/

# What this PR do:
- Adds IDS and TITLES to reference documentation so Docusaurus can find the pages
# How to test it
- Check the magic tunel link posted above -> Go to some component and try to refresh it in your browser, it should not throw 404 now
